modules/ROOT/pages/sharing-applications.adoc

```diff
@@ -0,0 +1,30 @@
+= Share Applications
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: sharing applications, sharing,
+
+When working on a team, a Mule runtime engine project increases in size, increases in its number of developers, and must run in other environments, such as test and production.
+
+Sharing applications is an opportunity for splitting work across teams, whether these teams work on the same overarching project or on different projects with an accent put on reuse.
+Applications are self-contained archives and can therefore be shared via a simple file or web server.
+
+To share applications follow the next steps:
+
+. You must document the name and content of the properties file that each application is expecting to use. +
+The administrator(s) who deploys and runs the Mule application can encrypt the properties file as detailed in xref:secure-configuration-properties.adoc[Secure Configuration Properties]. Administrators must keep the decryption key of the properties file secure.
+
+[start=2]
+. Generate the file to share the application by the following options :
+* Export the application from Anypoint Studio and generate a deployable archive `.jar` file.
+* Package your application using the Mule Maven Plugin and generate a:
+** Deployable archive `.jar` file to then deploy it into a running Mule runtime engine.
+** Mule app package `.jar` file to then import it into Studio.
+** Mule app lightweight package `.jar` file to then import it into Studio.
+
+[start=3]
+. After generating the `.jar` file, you can share it with other teams so they can continue working with the application.
+
+== See Also
+* xref:studio::import-export-packages.adoc#export-project-studio[Exporting Projects from Studio]
+* xref:mmp-concept#mule-application-packaging.adoc [Mule Application Packaging]
```

Comment:
this article was modified, please change the link to: 
* xref:package-a-mule-application[Mule Application Packaging]